phac-nml/irida-next
# enhancement: Update to use proper buttons CSS classses

## What does this PR do and why?
_Describe in detail what your merge request does and why._

1. Fixes buttons on groups details page, see below
2. Updates all buttons to use new css `buttton` and helper classes.
3. Removes old `btn` classses

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

Group details page before update:
![image](https://github.com/phac-nml/irida-next/assets/11295750/b2c8fba4-a201-4384-a134-ac856d0c9188)

Group details page after update:
![image](https://github.com/phac-nml/irida-next/assets/11295750/c512182c-e1a7-4ffd-830a-f5396d2396c2)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Visual inspection to make sure buttons are rendered properly
2. Especially look at group details page

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

  the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.
